Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/mysql/MySQLNode.java

```diff
@@ -50,7 +48,7 @@
     private ServerState serverState;
 
     public MySQLNode(AzureRefreshableNode parent, String subscriptionId, Server mysqlServer) {
-        super(mysqlServer.id(), mysqlServer.name(), parent, ICON_FILE, true);
+        super(mysqlServer.id(), mysqlServer.name(), parent, null, true);
```

Comment:
Remove the icon in my sql node?

Reply:
Yes. will set a AzureIcon for node on priority.